### DISCUSSION TITLE
select searchable property not working accordingly ?

### DISCUSSION THREAD

**Discussion by jichiduo** [wireui/wireui#223] [BUG]
when you use `:options="['Active', 'Pending', 'Stuck', 'Done']"` as the datasource, if you add searchable in the select component, the search field will not show  but if you use slot options  as datasource, even you add searchable="false" in the component , the search filed will show and works well
```html
    <x-select.option label="Pending" value="1" />
    <x-select.option label="In Progress" value="2" />
    <x-select.option label="Stuck" value="3" />
    <x-select.option label="Done" value="4" />
``` 

Im not sure this is a feature or a bug ?

**Comment by mfazail** [comment]
If you want to disable searching 
```
<x-select searchable="{{ false }}" >
    // your options
</x-select>
```
